doc/7-systems/es5503.md

```diff
@@ -0,0 +1,27 @@
+# Ensoniq ES5503 (ES5503)
+
+A sound chip designed by Robert "Bob" Yannes (the same guy who designed the [MOS Technology SID](c64.md)). It features 32 time-multiplexed channels (oscillators) of wavetable/sample audio.
```

Comment:
Just for consistency's sake with other documentation, perhaps start this and the following sentences with lowercase. (It's a weird quirk of tilde's writing style, but I'm not about to go through and "correct" everything to proper sentence case, so lowercase it is.)